### PR TITLE
Update README.md, add documentation

### PR DESCRIPTION
**Pull request by maria8ch**
This is for Issue #4

### CODE REVIEW

**Comment by maria8ch** [comment]
Almost done with the page, just need someone to look over it and help clarify certain steps.

**Comment by maria8ch** [comment]
Finished Issue #4 . Need reviews and a merge.

File: docs/index.md [hunk 1]
```diff
@@ -1,3 +1,45 @@
...
+* Collect test data of previously-acquired focus images.
+* Write python class to determine the average FWHM of an image (photutils?).
+* Write python class to fit a parabola to a set of points with uncertainties (scipy.optimize?).
+
+Putting it all together:
+* Write a script to take exposures while moving the focuser, fit PSFs to the images, and compute the optimum focus.
+* Generate plots to monitor the output of the above.
+
+
+My suggestion of a place to start would be to find a dozen or so images from a focus sequence at MRO, some above, some near, and some below focus. This will give us some test data to play with. We would also want the logbooks that go with those images, so that we can compare with the focus values measured by the observers.
```
Comment (darrenhunt2) [inline docs/index.md]: Maybe reword this line:

"Starting with 10 images at, above, and below focus...."

File: docs/index.md [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+
+I would next try to use the astropy affiliate photutils package to detect and measure point sources on each image and compute the mean full-width at half maximum (FWHM) for that image. One thing I don't know is how well photutils does on images that are fairly far from correct focus, such that the stars look more like donuts than points. This would be something to explore.
```
Comment (darrenhunt2) [inline docs/index.md]: Maybe: "Next, we will..."

File: docs/index.md [hunk 3]
```diff
@@ -1,3 +1,45 @@
...
+
+Width of Gaussian is not the same as FWHM
+
+Inside, outside (above, below) focus
+
+Use 3-6 + exposures points to fit a parabola, the minimum of the parabola is the optimum focus point! Plot FWHM in pixels vs focus steps in mm.
+
+Two ways: Measure FWHM and take the smallest value; look for the image the star is the brightest (small radius)
+
+Note: the FWHM method does not work for “donuts” -- extremely out of focus stars.
```
Comment (darrenhunt2) [inline docs/index.md:46]: The rest of this seems to look good/is pretty self explanatory. I think this will work otherwise.

**Review by darrenhunt2 — CHANGES_REQUESTED** [review]
Seems good other than the two comments that I left

**Review by darrenhunt2 — APPROVED** [review]

**Comment by parejkoj** [comment]
Don't worry, I'll get to my review...

File: docs/index.md [hunk 4]
```diff
@@ -1,3 +1,43 @@
...
+### Putting it all together
+* Write a script to take exposures while moving the focuser
+* Fit PSFs to the images
+* Compute the optimum focus
+* Generate plots to monitor the output of the above
+
+## Detailed explanation
+
+### Focus math
+* Starting with 10 images at, above, and below focus (taken from MRO with logbooks intact), with some above, some near, and some below focus. This will give us some test data to play with. Logbooks are necessary to compare with the focus values measured by the observers.
```
Comment (parejkoj) [inline docs/index.md]: The first sentence has duplication of the "above, near, below focus" bit.

File: docs/index.md [hunk 5]
```diff
@@ -1,3 +1,43 @@
...
+
+* Use the astropy affiliate photutils package to detect and measure point sources on each image, compute the mean full-width at half maximum (FWHM) for that image.
+
+* Write the code to do the focusing. This will be done before the automating part (that communicates with the telescope).
```
Comment (parejkoj) [inline docs/index.md]: "do the focusing": did you mean "compute the optimum focus from the set of images"?

File: docs/index.md [hunk 6]
```diff
@@ -1,3 +1,43 @@
...
+
+Our program needs to: 
+  * Pick a bright star in image, measure its FWHM (keep in mind that width of Gaussian is not the same as FWHM)
+  * Inside, outside (above, below) focus
```
Comment (parejkoj) [inline docs/index.md:41]: I think you meant here "take images at a range of different focuser positions"

File: docs/index.md [hunk 7]
```diff
@@ -1,3 +1,43 @@
-## The UW MRO automated focusing system
+# The UW MRO automated focusing system
 
-These pages will describe the automated focusing system being developed by students in UW Astronomy Undergrad Engineering Group (AUEG).
+These pages will describe the automated focusing system being developed by students in UW Astronomy Undergrad Engineering Group (AUEG). We are writing in python. There are two ways to achieve focusing:  
+a) measure FWHM and take the smallest value  
+b) look for the brightest pixels in the image to calculate radius of star and go from there
```
Comment (parejkoj) [inline docs/index.md]: The "brightest pixel" method does not require that you "calculate radius of star". It's based entirely on getting as much light focused onto a single pixel as possible. If you're going to mention it here, it would be good to also mention it below as something to check in the focus images, as a comparison with the star FWHM.

File: docs/index.md [hunk 8]
```diff
@@ -1,3 +1,43 @@
...
+
+## Here is a list of steps to take, in order:
```
Comment (parejkoj) [inline docs/index.md]: Shorter header please. Maybe something like: `## Steps to build automated focuser`?

**Review by parejkoj — CHANGES_REQUESTED** [review]
Thanks for collating the notes here.

Overall comment: If you put each sentence on its own line, it is easier to follow changes to those sentences. Markdown automatically merges sentences into paragraphs, so long as there is not an empty line between them.

File: docs/index.md (hunk 6 above)
Comment (maria8ch) [reply docs/index.md:41]: i changed some wording. is this good or should i just delete the two lines and replace with your quotation mark words?

**Review by maria8ch — COMMENTED** [review]
Replied to John's comments, changed some stuff. Are we good now?

**Review by parejkoj — APPROVED** [review]
Only other comment is "focus so that those pixels saturate": it's not so they saturate, just so that there is as much flux as possible in the brightest pixels.

**Comment by maria8ch** [comment]
ok so just so i'm clear- I changed the last little thing on the doc, so now I can click the green merge pull request button, right?

**Comment by parejkoj** [comment]
Your changes are approved by both reviewers: merge away!

And then we can see if the changes are propagated to the website...
